### DISCUSSION TITLE
Multi-Track Audio Output

### DISCUSSION THREAD

**Discussion by jbcurler2010** [DistroAV/DistroAV#890] [Q&A - Support]
It seems that the NDI Plugin (either from the Program NDI Output or Dedicated NDI Filter) only outputs Audio to Track 1. Is this correct or am I missing something? I would love to be able to manage all 6 OBS tracks on an NDI Stream. Thanks.

**Comment by Trouffman** [comment]
@jbcurler2010 Can you try again with the latest build and confirm if this is still a valid issue?
